src/main.ts

```diff
@@ -0,0 +1,8 @@
+import { NestFactory } from '@nestjs/core'
+import { AppModule } from './app/app.module'
+
+async function bootstrap() {
+    const app = await NestFactory.create(AppModule)
+    await app.listen(3000)
```

Comment:
remove? lets do like the discord sample: https://github.com/fjodor-rybakov/discord-nestjs/blob/master/sample/01-hello-bot-app/src/main.ts

Reply:
yeap